types/auto/modules/red/red.d.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * An encoder for RFC 2198 redundancy using WebRTC Insertable Streams.
+ */
+export class RFC2198Encoder {
```

Comment:
Why is this part of this PR?

Reply:
These were generated when I ran the build, didn't want gh to complain that the repo is not clean. This is going to be a separate commit anyways, so didn't see any harm in adding it to this PR.

Reply:
👍